tests/manual/placeholder.md

```diff
@@ -0,0 +1,8 @@
+## Editor placeholder
+
+1. Make sure the placeholder is visible in the editor without content.
+1. Focus the editor — the placeholder should disappear.
+1. Blur the editor — the placeholder should re–appear.
+1. Focus the editor with the content
+1. Remove the content, no placeholder should be visible.
+1. Blur the editor, the placeholder should appear.
```

Comment:
It would be cool to make sure in the manual sample that classes are properly reset, so some create and delete buttons which let me be sure that styles are restored on the element.

Reply:
Or can I test it on any already existing manual test?

Reply:
I added automated tests for that purpose in all ckeditor5-editor-* ([classic](https://github.com/ckeditor/ckeditor5-editor-classic/pull/79/commits/77bfcd494f99148c986ccb734591ce51b98bd5ef)).